.github/workflows/release.yaml

```diff
@@ -64,10 +61,10 @@ jobs:
       - name: Release
         uses: goreleaser/goreleaser-action@v2
         with:
-          version: v0.183.0
+          version: latest
```

Comment:
ditto

Reply:
to be able to use sign feature, I need to update the latest version of GoReleaser

Reply:
I got it. Please pin the version then. Otherwise, we have a concern about supply chain attacks and may face an unexpected error.

Reply:
gotcha! fixed right now, thanks